libr/debug/p/native/linux/linux_debug.c

```diff
@@ -1041,59 +1024,88 @@ static bool dump_this_map(FILE *f, ut64 start_addr, ut64 end_addr, bool file_bac
 	return false;
 }
 
+static char *read_alloc_from_file(FILE *f) {
```

Comment:
RDebug shouldt use FILE at all. there are apis in RUtil to read files that honor the sandbox